From a838254a8248df9644505f28ce091b410ac19031 Mon Sep 17 00:00:00 2001
From: szecsi <szecsi.laszlo@gmail.com>
Date: Tue, 9 Feb 2021 19:10:23 +0100
Subject: [PATCH] triangle geometry draw

---
 client/src/main/kotlin/TriangleGeometry.kt | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/client/src/main/kotlin/TriangleGeometry.kt b/client/src/main/kotlin/TriangleGeometry.kt
index 885d2cc..0887b21 100644
--- a/client/src/main/kotlin/TriangleGeometry.kt
+++ b/client/src/main/kotlin/TriangleGeometry.kt
@@ -41,4 +41,12 @@ class TriangleGeometry(val gl : WebGL2RenderingContext) {
     gl.bindVertexArray(null)
   }
 
+  fun draw() {
+
+    gl.bindVertexArray(inputLayout)
+    gl.bindBuffer(GL.ELEMENT_ARRAY_BUFFER, indexBuffer)  
+
+    gl.drawElements(GL.TRIANGLES, 3, GL.UNSIGNED_SHORT, 0) //#3# pipeline is all set up, draw three indices worth of geometry
+  }
+
 }
-- 
2.16.1.windows.1

