From 8f8e9ebdb6263864ce112cae295cf5d2f7872081 Mon Sep 17 00:00:00 2001
From: szecsi <szecsi.laszlo@gmail.com>
Date: Thu, 11 Feb 2021 18:35:13 +0100
Subject: [PATCH] Calculate time step

---
 client/src/main/kotlin/Scene.kt | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/client/src/main/kotlin/Scene.kt b/client/src/main/kotlin/Scene.kt
index ee658e3..bfa18e6 100644
--- a/client/src/main/kotlin/Scene.kt
+++ b/client/src/main/kotlin/Scene.kt
@@ -1,5 +1,6 @@
 import org.w3c.dom.HTMLCanvasElement
 import org.khronos.webgl.WebGLRenderingContext as GL //# GL# we need this for the constants declared ˙HUN˙ a constansok miatt kell
+import kotlin.js.Date
 
 class Scene (
   val gl : WebGL2RenderingContext){
@@ -13,8 +14,16 @@ class Scene (
     gl.viewport(0, 0, canvas.width, canvas.height)//#viewport# tell the rasterizer which part of the canvas to draw to ˙HUN˙ a raszterizáló ide rajzoljon
   }
 
+  val timeAtFirstFrame = Date().getTime()
+  var timeAtLastFrame =  timeAtFirstFrame
+
   @Suppress("UNUSED_PARAMETER")
   fun update(gl : WebGL2RenderingContext, keysPressed : Set<String>) {
+    val timeAtThisFrame = Date().getTime() 
+    val dt = (timeAtThisFrame - timeAtLastFrame).toFloat() / 1000.0f
+    val t  = (timeAtThisFrame - timeAtFirstFrame).toFloat() / 1000.0f    
+    timeAtLastFrame = timeAtThisFrame
+    
     gl.clearColor(0.3f, 0.0f, 0.3f, 1.0f)//## red, green, blue, alpha in [0, 1]
     gl.clearDepth(1.0f)//## will be useful in 3D ˙HUN˙ 3D-ben lesz hasznos
     gl.clear(GL.COLOR_BUFFER_BIT or GL.DEPTH_BUFFER_BIT)//#or# bitwise OR of flags
-- 
2.16.1.windows.1

