From 26ed76d0812310c5a6354e58ab4a361dd7394449 Mon Sep 17 00:00:00 2001
From: szecsi <szecsi.laszlo@gmail.com>
Date: Tue, 23 Feb 2021 14:37:16 +0100
Subject: [PATCH] GameObject update

---
 client/src/main/kotlin/GameObject.kt | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/client/src/main/kotlin/GameObject.kt b/client/src/main/kotlin/GameObject.kt
index 680a2ec..35ef215 100644
--- a/client/src/main/kotlin/GameObject.kt
+++ b/client/src/main/kotlin/GameObject.kt
@@ -17,4 +17,12 @@ class GameObject(
   init { 
     addComponentsAndGatherUniforms(*meshes)
   }
+
+  fun update() {
+    modelMatrix.set().
+      scale(scale).
+      rotate(roll).
+      translate(position)
+  }
+
 }
-- 
2.28.0.windows.1

